api.md

```diff
@@ -170,6 +172,66 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## attach_data_node() [](attach_data_node)
+
+Attach a data node to a hypertable. The data node should have been
+previously created using [`add_data_node`](#add_data_node).
+
+When a distributed hypertable is created it will by default use all
+available data nodes for the hypertable, but if a data node is added
+*after* a hypertable is created, the data node will not automatically
+be used by existing distributed hypertables.
+
+If you want a hypertable to use a data node that was created later,
+you need to attach the data node to the hypertable using this
+function. This function *MUST* be run on an access node.
+
+#### Required Arguments [](attach_data_node-required-arguments)
+
+| Name              | Description                                   |
```

Comment:
All of your tables need the `|--|--|` lines to designate them as tables

Reply:
Done